### ISSUE TITLE
Timer notification received two times 

### ISSUE THREAD

**Issue by mjalil** [camunda/camunda-bpm-reactor#93]
Hi dear developers, 
I registered for every `start` event on a bpmn that have an intermediate timer task and implemented `ExecutionListener` on a class. Then I see timer start event received 2 times. Is it my fault or a bug?
```java
SelectorBuilder.selector()
    .context(SelectorBuilder.Context.bpmn)
    .event("start");
```
Thanks

**Comment by jangalinski** [comment]
Just for clarification: you are talking about timer-start-events (intermediate timer task and start event doesn't fit).

**Comment by mjalil** [comment]
yes, the first event is received when execution reaches the timer 'timer-start-event'. second 'start' event comes when the duration of that timer exceed. I need the second one but both of them raises and the same as each other and I cannot distinguish between them

**Comment by jangalinski** [comment]
could you provide a PR with a failing test example for this, please?
